docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -177,6 +177,7 @@ Due to the code cleanup, the following deprecated items have been dropped:
 |`SearchResultsItemComponent`|-|`ContentTableItemComponent`|
 |`onlyContentOnTheFly`|-|`visibleTabs`|
 |`cotfForcedLanguage`|-|`cotfAllowedLanguages`|
+|`ContentTypeNames`|ezplatform-admin-ui|`getContentTypeName`|
```

Comment:
I would rather go with
|`contentTypeNames`|`window.eZ.adminUiConfig`|`contentTypes`|

and inform about new helper method (`window.eZ.helpers.contentType.getContentTypeName`) somewhere else.

Reply:
Information about new helper will be added in the Release Notes, as it is something new.